File: my-submissions/1476-count-negative-numbers-in-a-sorted-matrix/solution.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
var countNegatives = function(grid) {
    let count = 0;

    for(const row of grid){
        for(const num of row){
            count += (num < 0);
        }
    }

    return count;
};

File: my-submissions/1476-count-negative-numbers-in-a-sorted-matrix/solution.java
class Solution {
    public int countNegatives(int[][] grid) {
        int count = 0;

        for(int[] row : grid) {
            for(int num : row) {
                count += (num < 0) ? 1 : 0;
            }
        }

        return count;
        
    }
}
